docs/getting-started.md

```diff
@@ -139,6 +139,17 @@ Now that you're setup, if you can run it on the command line, you can run it in
     4. For `Bazel Command` you can put any bazel command, like `build`, `run`, `test`, etc.
     5. Click `Ok`, then there should be a green arrow in the top right corner by the drop-down menu. Click it and the test will run!
 
+### Running our AI, Simulator or Robot Diagnostics
```

Comment:
Can we add a description of what running AI/Simulator/Robot Diagnostics does? Like why would someone want to do this?

Reply:
added